packages/worker/src/replay.ts

```diff
@@ -0,0 +1,10 @@
+export interface ReplayResults {
+  // True if any workflow failed replay
+  readonly hadAnyFailure: boolean;
+  // Maps run id to information about the replay failure
+  readonly failureDetails: Map<string, Error>;
```

Comment:
Is runId going to be enough to debug the error source? We should probably add the workflowId here too.
I'd change this to:
```
interface WorkflowRunError { workflowId: string; runId: string; error: Error }
readonly failureDetails: Array<WorkflowRunError>
```